frontend/src/metabase/admin/settings/components/widgets/GroupMappingsWidget/GroupMappingsWidget.jsx

```diff
@@ -22,40 +26,36 @@ import {
 } from "./GroupMappingsWidget.styled";
 import MappingRow from "./MappingRow";
 
-const groupIsMappable = group => !isDefaultGroup(group);
-
-const loadGroupsAndMappings = async mappingSetting => {
-  const [settings, groupsIncludingDefaultGroup] = await Promise.all([
-    SettingsApi.list(),
-    PermissionsApi.groups(),
-  ]);
-
-  const setting = _.findWhere(settings, {
-    key: mappingSetting,
-  });
-
+const mapStateToProps = (state, props) => {
```

Comment:
Let's have a container component for GroupMappingsWidget to separate the layer that interacts with the store. You can have a look at `frontend/src/metabase/home/homepage/containers/HomeContent/HomeContent.tsx` as an example